src/appleseed/renderer/kernel/shading/shadingcontext.h

```diff
@@ -132,6 +132,12 @@ class ShadingContext
         const ShaderGroup&  shader_group,
         const ShadingPoint& shading_point) const;
 
+    void execute_osl_matte(
+        const ShaderGroup&      shader_group,
```

Comment:
Align arguments vertically. While you're at it, maybe you can do the same for `execute_osl_npr()`'s arguments.